projects/plugins/mu-wpcom-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.11-beta - 2023-03-20
```

Comment:
Looks like we are releasing a beta of the mu-wpcom-plugin here. We can just release the next stable version, 1.0.11.